mod_reforged/hooks/ai/tactical/agent.nut

```diff
@@ -0,0 +1,20 @@
+::mods_hookNewObject("ai/tactical/agent", function (o) {
+	local addBehavior = o.addBehavior;
+	o.addBehavior = function( _behavior )
+	{
+		local className;
```

Comment:
Why is this variable declared outside?
It looks like it's used only a single time during the if. You might not even need to use a variable at all here

Reply:
Done.